pootle/apps/pootle_app/models/directory.py

```diff
@@ -56,6 +56,12 @@ class Directory(models.Model, CachedTreeItem):
     # see migration 0005_case_sensitive_schema.py
     pootle_path = models.CharField(max_length=255, null=False, db_index=True,
                                    unique=True, default='/')
+    tp = models.ForeignKey(
+        'pootle_translationproject.TranslationProject',
+        related_name='dirs',
+        null=True,
+        blank=True,
+        db_index=True)
```

Comment:
Django 1.10 needs `on_delete` to be explicitly see cf7907914f991180161463732b08b4e6d176591d.